target/linux/ipq40xx/patches-5.4/712-add-port-isolation-flag.patch

```diff
@@ -0,0 +1,13 @@
+--- a/net/Kconfig
++++ b/net/Kconfig
+@@ -253,6 +253,10 @@ source "net/l3mdev/Kconfig"
+ source "net/qrtr/Kconfig"
+ source "net/ncsi/Kconfig"
+ 
++config IPQ40XX_PORT_ISOLATION
```

Comment:
Why is this here instead of under ESSEDMA?

Reply:
You mean `drivers/net/ethernet/qualcomm/Kconfig`? I can put it there.

Reply:
like this?

Reply:
Yes, but I would also add the help text explaining what it is used for.
I personally would even drop the IPQ40xx specific prefix.

Reply:
Okay, I will add some helptext. :) Is my commit message correct?
Suddenly, `target/linux/install` fails.  I will make a break and debug this.
Thanks.

Reply:
I added some helptext. I hope this is enough.

Reply:
---help--- has been deprecated for a while and newer kernel versions will actually error out on these.
So a simple help is preferred, I have changed this ---help--- to help in the PHY PR.

Reply:
Thanks. Changed.